CONTRIBUTING.md

```diff
@@ -1,32 +1,32 @@
-# Contributing to Contributivity
+# Contributing to mplc
 
-> Please, feel free to suggest any other relevant item, to ask for edits or even to submit a Pull Request!
->
-> You can also have a look at these broader [contributing guidelines](https://github.com/SubstraFoundation/.github/blob/master/CONTRIBUTING.md).
+> *Please, feel free to suggest any other relevant item, to ask for edits or even to submit a Pull Request!*
+> *You can also have a look at these broader [contributing guidelines](https://github.com/SubstraFoundation/.github/blob/master/CONTRIBUTING.md)*.
 
-- [Contributing to Contributivity](#contributing-to-contributivity)
-  - [1. Git workflow & branching](#1-git-workflow--branching)
-  - [2. Python](#2-python)
-    - [2.1 Python Virtual Environment](#21-python-virtual-environment)
-    - [2.2 Python Enhancement Proposals (PEP)](#22-python-enhancement-proposals-pep)
+Table of content:
+
+1. [Git workflow & branching](#1-git-workflow--branching)
+1. [Python](#2-python)
+   1. [Python Virtual Environment](#2i-python-virtual-environment)
+   1. [Python Enhancement Proposals (PEP)](#2ii-python-enhancement-proposals-pep)
       - [Black formatter](#black-formatter)
-        - [Installation](#installation)
-        - [Usage](#usage)
       - [Flake8 linter](#flake8-linter)
-    - [2.3 Basic module structure & Imports order](#23-basic-module-structure--imports-order)
-    - [2.4 Jupyter Notebooks](#24-jupyter-notebooks)
-      - [2.5 Sharing & online rendering](#25-sharing--online-rendering)
-  - [3. Further Resources](#3-further-resources)
+   1. [Basic module structure & Imports order](#2iii-basic-module-structure--imports-order)
+   1. [Jupyter Notebooks](#2iv-jupyter-notebooks)
+   1. [Sharing & online rendering](#2v-sharing--online-rendering)
+1. [Further Resources](#3-further-resources)
 
 ## 1. Git workflow & branching
 
-As long as other people contribute to a repository, it is easier and safer to restrain direct actions on the `master` branch. To do so, it is advised to create new branches, based on `master` (or a dedicated sub-branch, `dev` for example), to develop any new feature and then to open a Pull Request. Once audited and validated, it can be merged into `master`, and so on.
+The branching model of the project is a very simplified of the [standard approach](https://nvie.com/posts/a-successful-git-branching-model/):
 
-To go further, here is a good example of a successful project build with a versioning file system (git, mercurial, etc.): <https://nvie.com/posts/a-successful-git-branching-model/>
+- The `master` branch is protected
+- Contributors create a feature branch from `master`, push it to the repository and open a draft PR
```

Comment:
In fact if you are not added to the contributor list, you'll have to fork the repo and create your PR from it, is it worth mentioning it?

Reply:
In this case, I would just add a note saying: "> Note: If you are not yet part of the contributor list, please get in touch with us. In the meantime, you can create your own fork of the project and open a PR pointing to this source repository, so we can evaluate your submission. [Github documentation](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork)."